packages/components/organisms/f-checkout/test-utils/component-objects/f-checkout.component.js

```diff
@@ -124,20 +123,12 @@ module.exports = class Checkout extends Page {
      * @param {String} checkout.isValid The checkout validation
      */
 
-    load (componentToWaitFor='default') {
-        const pageUrl = buildUrl(this.componentType, this.componentName, this.path);
-        this.open(pageUrl);
-
-        switch (componentToWaitFor) {
-            case 'error':
-                this.waitForErrorPageComponent();
```

Comment:
Is the method below still required? If not you can get rid of it.

Reply:
load()? That one is in use.

Reply:
👍🏻